CMakeLists.txt

```diff
@@ -201,14 +201,19 @@ IF(WANT_LV2)
 	IF(PKG_CONFIG_FOUND)
 		PKG_CHECK_MODULES(LV2 lv2)
 		PKG_CHECK_MODULES(LILV lilv-0)
+	ENDIF()
+	IF(NOT LV2_FOUND AND NOT LILV_FOUND)
+		FIND_PACKAGE(LV2 CONFIG)
+		FIND_PACKAGE(LILV CONFIG)
 		IF(LV2_FOUND AND LILV_FOUND)
-			SET(LMMS_HAVE_LV2 TRUE)
-			SET(STATUS_LV2 "OK")
-		ELSE()
-			SET(STATUS_LV2 "not found, install it or set PKG_CONFIG_PATH appropriately")
+			SET(LILV_LIBRARIES "lilv::lilv")
```

Comment:
You don't have to check for `LV2_FOUND` here, right?

Reply:
Good point, will fix. 👍

Reply:
Fixed, and rebased on master while I was at it.